modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -38,7 +38,9 @@ public struct Color : IEquatable<Color>
         public float B;
 
         /// <summary>
-        /// The color's alpha (transparency) component, typically on the range of 0 to 1.
+        /// The color's alpha (opacity) component, typically on the range of 0 to 1.
```

Comment:
Core documentation doesn't include the parenthesized word.

```suggestion
        /// The color's alpha component, typically on the range of 0 to 1.
```